ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java

```diff
@@ -1511,9 +1525,10 @@ public static void mvFileToFinalPath(Path specPath, Configuration hconf,
     //      Deduplicate and keep a list of files
     //      Pass on the list of files to conf (to be used later by fetch operator)
     //    if true:
-    //      Rename tmpPath to a new directory name
-    //      Remove duplicates
-    //      Rename/move the new directory to specPath
+    //       1) Rename tmpPath to a new directory name to prevent additional files
```

Comment:
can we also restore the comment which already existed?

Reply:
@ashutoshc the earlier comment does not make sense anymore. for blobstore FS there is no step number 2. What comment line are you referring to?